### ISSUE TITLE
ServiceControl 3x does not include all of the RabbitMQ dependencies

### ISSUE THREAD

**Issue by boblangley** [Particular/ServiceControl#1502]
The RabbitMQ client has a dependency on [`Microsoft.Diagnostics.Tracing.EventSource.Redist`](https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing.EventSource.Redist/1.1.28) which is not being packaged in the 3.x branch. This is probably a new dependency of the transport from the move to core v7.

The packaging project does not include it: https://github.com/Particular/ServiceControl/blob/develop/src/ServiceControlInstaller.Packaging/ServiceControlInstaller.Packaging.csproj#L57-L59

When this is not included [it can result in RabbitMq being unable to properly raise/log exceptions](https://groups.google.com/forum/#!topic/rabbitmq-users/JuC04MXYbmI):

```
Exception Info: System.IO.FileNotFoundException
at RabbitMQ.Client.ESLog.Error(System.String, System.Exception)
at RabbitMQ.Client.Framing.Impl.Connection.LogCloseError(System.String, System.Exception)
at RabbitMQ.Client.Framing.Impl.Connection.MainLoop()
at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
at System.Threading.ThreadHelper.ThreadStart()
```

**Comment by andreasohlund** [comment]
Just adding that the monitoring is packaging *.dll so it wouldn’t be effected

**Comment by SzymonPobiega** [comment]
Fixed in #1521
